### PR TITLE
bugs.fd.o bug archelogy

### PR DESCRIPTION
**Pull request by keszybz**
Three unrelated fixes for old (but still valid) issues.

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM.

File: man/systemctl.xml [hunk 1]
```diff
@@ -692,6 +692,42 @@
...
 
+            <para>Produces output similar to
+            <programlisting>  UNIT                         LOAD   ACTIVE SUB     DESCRIPTION
+  sys-module-fuse.device       loaded active plugged /sys/module/fuse
+  -.mount                      loaded active mounted Root Mount
+  boot-efi.mount               loaded active mounted /boot/efi
+  systemd-journald.service     loaded active running Journal Service
+  systemd-logind.service       loaded active running Login Service
+● user@1000.service            loaded active running User Manager for UID 1000
+...
```
Comment (poettering) [inline man/systemctl.xml:703]: should probably be unicode … rather than three separate dots…

File: man/systemctl.xml [hunk 2]
```diff
@@ -692,6 +692,42 @@
...
+SUB    = The low-level unit activation state, values depend on unit type.
+
+123 loaded units listed. Pass --all to see loaded but inactive units, too.
+To show all installed unit files use 'systemctl list-unit-files'.
+            </programlisting>
+            The header and the last unit of a given type are underlined if the
+            terminal supports that. A colored dot is shown next to services which
+            were masked, not found, or otherwise failed.</para>
+
+            <para>The LOAD column shows the load state, one of
```
Comment (poettering) [inline man/systemctl.xml:717]: not that it matters, but LOAD and the other column names should probably be in `<literal>`

File: man/systemctl.xml (hunk 1 above)
Comment (keszybz) [reply man/systemctl.xml:703]: Ack.

File: man/systemctl.xml (hunk 2 above)
Comment (keszybz) [reply man/systemctl.xml:717]: I think the capital letters in LOAD make it distinguishable enough — adding quotes around that would be like underlining and italics, just too much.

**Pull request merged**